signing_service/setup.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import os
+from setuptools import setup
+
+
+def get_version():
+    with open(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'RELEASE-VERSION',
+        ),
+        'r'
+    ) as version_file:
+        return version_file.read()
+
+
+setup(
+    name='Signing-Service',
+    version=get_version(),
+    url='',
+    maintainer='',
+    maintainer_email='',
```

Comment:
These fields need to be filled too.

Reply:
Right, but with what values ?

Reply:
- URL of the repository
- Code Poets as maintainer

Reply:
:+1: done